src/prisma-service.ts

```diff
@@ -0,0 +1,956 @@
+import { PrismaClient } from '@prisma/client';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import {
+  extractPagination,
+  Pagination,
+  extractTermParam
+} from './request-params';
+import * as TE from 'fp-ts/TaskEither';
+import { pipe } from 'fp-ts/lib/function';
+import { Prisma } from '@prisma/client';
+import {
+  balanceResponse,
+  dagBlockResponse,
+  dagTransactionResponse,
+  dagTransactionsResponse,
+  globalSnapshotResponse,
+  globalSnapshotsResponse,
+  handleError,
+  metagraphBlockResponse,
+  metagraphFeeTransactionResponse,
+  metagraphFeeTransactionsResponse,
+  metagraphSnapshotResponse,
+  metagraphSnapshotsResponse,
+  metagraphsResponse,
+  metagraphTransactionsResponse,
+  metagraphTransactionResponse,
+  missingParameterResponse,
+  notFoundResponse,
+  respond,
+  rewardsResponse
+} from './response';
+import { maxSizeLimit, SearchDirection } from './pagination';
+
+const prisma = new PrismaClient();
+
+const global_snapshot_exists = async (term) => {
```

Comment:
Stick w/camelcase for consistency
```suggestion
const globalSnapshotExists = async (term) => {
```